plugins/auth-jwt/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@amplication/plugin-auth-jwt",
-  "version": "1.4.0",
+  "version": "1.4.1-beta.4",
```

Comment:
how come from 1.4.0 we jumped to 1.4.1-beta.4? I think it should be 1.4.1-beta.1

Reply:
@abrl91  After approve, I will update the version to 1.4.1